### ISSUE TITLE
coq-mathcomp-finmap package 1.3.4 and 1.4.0 are incompatible with Coq 8.11.1

### ISSUE THREAD

**Issue by anton-trunov** [coq/opam-coq-archive#1230]
Here is the error message I get:

```
File "./finmap.v", line 831, characters 0-154:
Error: Notation "[ fset[ _ ] _ | _ : _ , _ : _ ]" is already defined
at level 0 with arguments constr Unknown level, constr at level 99, constr
at level 99, constr at level 200, constr at level 99, constr Unknown level
while it is now required to be at level 0 with arguments constr
Unknown level, constr at level 99, constr at level 99, constr Unknown level,
constr at level 99, constr Unknown level.
```

Notice that it compiles just fine with Coq 8.11.0.

/cc @CohenCyril

**Comment by anton-trunov** [comment]
It was also reported here: https://github.com/math-comp/finmap/issues/62

**Comment by anton-trunov** [comment]
Fixed in #1231
